Tests/NIOSSLTests/SecureBytesTests.swift

```diff
@@ -0,0 +1,228 @@
+        aDifferentMiddleIndex = copy.index(aMiddleIndex, offsetBy: 5)
+        copy.replaceSubrange(aMiddleIndex..<aDifferentMiddleIndex, with: [1, 2, 3, 4, 5])
+        XCTAssertEqual(copy.count, 10)
+        XCTAssertEqual(Array(copy), [0, 0, 1, 2, 3, 4, 5, 0, 0, 0])
+        XCTAssertEqual(Array(base), baseBytes)
+        XCTAssertNotEqual(copy, base)
+
+        // The fifth is to make the storage bigger.
+        copy = base
+        aMiddleIndex = copy.index(copy.startIndex, offsetBy: 2)
+        aDifferentMiddleIndex = copy.index(aMiddleIndex, offsetBy: 5)
+        copy.replaceSubrange(aMiddleIndex..<aDifferentMiddleIndex, with: [1, 2, 3, 4, 5, 6, 7])
+        XCTAssertEqual(copy.count, 12)
+        XCTAssertEqual(Array(copy), [0, 0, 1, 2, 3, 4, 5, 6, 7, 0, 0, 0])
+        XCTAssertEqual(Array(base), baseBytes)
+        XCTAssertNotEqual(copy, base)
+    }
+
+    func testResizingByMakingLarger() {
+        var base = SecureBytes(count: 12)
+        XCTAssertGreaterThanOrEqual(base.backing.capacity, 16)
+        XCTAssertEqual(base.count, 12)
+
+        base.append(contentsOf: 0..<16)
+        XCTAssertGreaterThanOrEqual(base.backing.capacity, 32)
+        XCTAssertEqual(base.count, 28)
+
+        base.append(contentsOf: 0..<4)
+        XCTAssertGreaterThanOrEqual(base.backing.capacity, 32)
+        XCTAssertEqual(base.count, 32)
+    }
+
+    func testCountInitializerGeneratesSomewhatRandomData() {
+        let base = SecureBytes(count: 16)
+        XCTAssertGreaterThanOrEqual(base.backing.capacity, 16)
+        XCTAssertEqual(base.count, 16)
+        XCTAssertNotEqual(Array(repeating: UInt8(0), count: 16), Array(base))
+    }
+
+    func testBackingBytesAreAppropriatelySized() {
+        var base = SecureBytes(repeating: 0, count: 10)
+        XCTAssertGreaterThanOrEqual(base.backing.capacity, 16)
+
+        base.withUnsafeBytes { XCTAssertEqual($0.count, 10) }
+        base.withUnsafeMutableBytes { XCTAssertEqual($0.count, 10) }
+        base.backing._withVeryUnsafeMutableBytes { XCTAssertGreaterThanOrEqual($0.count, 16) }
+    }
+
+    func testTheresOnlyOneRegion() {
+        var base = SecureBytes()
+        base.append(Data("hello".utf8))
+        base.append(Data("world".utf8))
+        XCTAssertEqual(base.regions.count, 1)
+    }
+
+    func testScaryInitializer() {
+        let base = SecureBytes(unsafeUninitializedCapacity: 5) { (scaryPointer, initializedCapacity) in
+            XCTAssertEqual(scaryPointer.count, 5)
+            scaryPointer.storeBytes(of: UInt32(0x01020304).bigEndian, as: UInt32.self)
+            initializedCapacity = 4
+        }
+
+        XCTAssertGreaterThanOrEqual(base.backing.capacity, 8)
+        XCTAssertEqual(Array(base), [1, 2, 3, 4])
+
+        func testThrowingInitialization() throws {
+            _ = try SecureBytes(unsafeUninitializedCapacity: 5) { (_, _) in
+                throw CryptoKitError.incorrectKeySize
+            }
+        }
```

Comment:
`scripts/generate_linux_tests.rb` doesn't seem to handle inline functions correctly. I'm not quite sure how much work it would be to correctly detect inline functions an skip them during.
For now, you should be able to just rewrite your inline function as a closure:
```suggestion
        let testThrowingInitialization: () -> throws () = {
            _ = try SecureBytes(unsafeUninitializedCapacity: 5) { (_, _) in
                throw CryptoKitError.incorrectKeySize
            }
        }
```

Reply:
Thank you @dnadoba!  Looks like I have it all fixed up now in cd37a41